packages/teleport-project-generator-angular/src/index.ts

```diff
@@ -0,0 +1,98 @@
+import { createPlugin as createAngularModuleGenerator } from '@teleporthq/teleport-plugin-module-generator-angular'
```

Comment:
is this a plugin or a generator?

Reply:
plugin, let me remove generator to elimintate the confusion